src/main/java/com/gachigang/ontherun/common/ApplicationConstants.java

```diff
@@ -24,6 +28,21 @@ public class Validation {
         public static final Pattern PASSWORD_PATTERN = Pattern.compile(Validation.PASSWORD_REGEX);
     }
 
+    /**
+     * Utility class for containing spring properties keys from `*.properties` files.
+     */
+    @UtilityClass
+    public class SpringProperties {
```

Comment:
I guess 'MessageProperties' or 'ErrorMessages' will be more convinient for the name, because from seeing SpringProperties it is hard to clearly understand that it is our custom error messages, it is more similar to Spring configuration properties witch came from Spring. So it is a little bit confusing

Reply:
I forget to delete this UtilityClass because we don't need it